### ISSUE TITLE
[BUG] Label order

### ISSUE THREAD

**Issue by JosefBuchner** [BrainLesion/GlioMODA#6]
### Describe the bug
Due to the way nnUNet saves the labels, the order of our original input labels gets changed.

### Steps to Reproduce
This is the label handler in nnUNet: 
`if self.has_regions:
            if isinstance(predicted_probabilities, np.ndarray):
                segmentation = np.zeros(predicted_probabilities.shape[1:], dtype=np.uint16)
            else:
                # no uint16 in torch
                segmentation = torch.zeros(predicted_probabilities.shape[1:], dtype=torch.int16,
                                           device=predicted_probabilities.device)
            for i, c in enumerate(self.regions_class_order):
                segmentation[predicted_probabilities[i] > 0.5] = c`

Therefore, each subsequent label overwrites the last. In our case, WT gets 1, TC 2 and ET 3. After stacking these labels, Edema = 1, Necrosis = 2 and ET = 3. Our original input has Edema as 2 and Necrosis as 1.

**Suggestion**
We either have to add a script to change the labels back to the original input labels or recommend an additional parameter for nnUNet.

**Comment by JosefBuchner** [comment]
It seems like regions_class_order is already used for that, therefore we only have to adjust that.
